homeassistant/components/habitica/util.py

```diff
@@ -159,32 +139,32 @@ def get_attribute_points(
     }
 
     equipment = sum(
-        stats[attribute]
+        getattr(stats, attribute)
```

Comment:
how else do you access a class attribute?

Reply:
Oh, there is a built-in getattr function for classes

Reply:
My impression is this is used to get "Int" and "Str" on a bunch of different objects. You may want to just invert this so there is a function to get Int from a bunch of objects and a function to get Str from a bunch of objects since it may be equivalent amount of code.

Reply:
attributes intelligence, strength, perception and constitution for 8 different gear types and i have to look up the equipped gears attribute modifiers from content, which contains hundreds of different gear items. Really sure I should unravel this and duplicate the code 4 times?

Reply:
Ah sorry, I only saw two in the diff but i can see there are more.

Reply:
You may consider doing this inside the library. (i'm not commenting on when to do this)